testing/TestHarness/TestHarness/Ext/Navigation/PageNavigation/PageNavigationOnePage.xaml

```diff
@@ -36,6 +36,9 @@
 					Click="{x:Bind ViewModel.SettingsWriteTest}"
 					Content="Settings Write Test" />
 			<TextBlock x:Name="TxtUrl" AutomationProperties.AutomationId="OnePageTxtUrlFromBrowser" />
+			<TextBlock x:Name="TxtDataContext"
+					   Text="DataContext is ok"
+					   AutomationProperties.AutomationId="OnePageTxtDataContext" />
```

Comment:
I feel like it would be simpler to just have `Text="{Binding }"`. Which should set the text to the fully qualified name of the DataContext's type. It should handle DataContextChanged properly with this method, I _think_.

Reply:
The issue was that the `FrameView` was initially propagating the wrong `DataContext` to its pages when each page instance was created. Since the pages are children of `FrameView`, they inherited this incorrect `DataContext`, which was then replaced by the correct one.

Using `Text="{Binding}"` would be appropriate if we were simply validating that the correct `DataContext` was eventually set. But, we need to ensure that the `DataContext` was never set to anything other than the expected `ViewModel` from the beginning.